src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/defendant/DefendantTrialArrangementsNotificationScenarioTest.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.civil.controllers.dashboard.scenarios.defendant;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.http.HttpStatus;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import uk.gov.hmcts.reform.civil.controllers.BaseIntegrationTest;
+import uk.gov.hmcts.reform.dashboard.data.ScenarioRequestParams;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications.DashboardScenarios.SCENARIO_AAA6_CP_TRIAL_ARRANGEMENTS_REQUIRED_DEFENDANT;
+
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@Testcontainers
+public class DefendantTrialArrangementsNotificationScenarioTest extends BaseIntegrationTest {
+
+    @Test
+    void shouldCreateAddTrialArrangementsForDefendant() throws Exception {
+        HashMap<String, Object> map = new HashMap<>();
+        map.put("respondent1PartyName", "Mr Defendant");
+        UUID caseId = UUID.randomUUID();
+        doPost(
```

Comment:
doPost is the old way of doing it.
We now call the handler directly, like in src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/claimant/CCJRequestedScenarioTest.java

So if you could update that :)

Reply:
I ask you about that changes and now I miss it :)